pkgs/top-level/all-packages.nix

```diff
@@ -5668,7 +5668,8 @@ with pkgs;
     ocamlPackages = ocaml-ng.ocamlPackages_4_14;
   };
 
-  inherit (coqPackages) compcert;
+  compcert = if lib.versionAtLeast coqPackages.coq.version "9.0" then null
+    else coqPackages.compcert;
```

Comment:
This change looks wrong.

Reply:
It's due to not having compcert on 9.0 yet. Alternatively we could keep 8.20 the default until compcert supports it.

Reply:
We could also write `coqPackages.compcert or null` I guess which would be more robust.

Reply:
Please just use `inherit (coqPackages_8_20) compcert;`

Reply:
Indeed, done, CI green again both here and on https://github.com/coq-community/coq-nix-toolbox/pull/335